account_receipt_base/views/account_move_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<odoo>
+    <record id="view_move_form_inherit" model="ir.ui.view">
+        <field name="name">Add Receipts to Account Move Form View</field>
+        <field name="model">account.move</field>
+        <field name="type">form</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='invoice_outstanding_credits_debits_widget']" position="attributes">
+                <attribute name="attrs">{'invisible':[('state', '!=', 'posted')]}</attribute>
+            </xpath>
```

Comment:
Nella vista ereditata `attrs` ha esattamente lo stesso valore:
https://github.com/odoo/odoo/blob/750d8d939b7e36cf2be810ed450e28827e871a51/addons/account/views/account_move_views.xml#L1038
potresti chiarire che effetto dovrebbe avere questa modifica?

Reply:
Ciao @SirAionTech , su 14.0 lo stato attuale mi risulta questo:

https://github.com/odoo/odoo/blob/6b9b598fbb29d0bd22172dd346f9b36fae43e0bd/addons/account/views/account_move_views.xml#L941-L944

Reply:
> Ciao @SirAionTech , su 14.0 lo stato attuale mi risulta questo:
> 
> https://github.com/odoo/odoo/blob/6b9b598fbb29d0bd22172dd346f9b36fae43e0bd/addons/account/views/account_move_views.xml#L941-L944

Hai ragione stavo guardando `16.0`, come non detto.

Ho visto che in `16.0` hanno fatto questa stessa modifica insieme ad altre per visualizzare i messaggi di outstanding credit/debit in https://github.com/odoo/odoo/commit/ed652d6c0e1bd68eb92c4c8c56e25647637cc600, @LorenzoC0 (cc @eLBati) volete aggiungerle?
Altrimenti per me si può mergiare già così.

Reply:
> Per me è ok 👍🏻

Va bene, allora fammi sapere quando hai fatto così possiamo procedere.
Se vuoi nel frattempo puoi mettere la PR in bozza (https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft) così evitiamo merge precoci.

Reply:
Scusami, mi sono espresso male, per me non sono necessarie ulteriori modifiche e possiamo unire il branch 👍🏻